### ISSUE TITLE
Have a minimal working skeleton based on Next.js

### ISSUE THREAD

**Issue by bfabio** [italia/developers.italia.it#1473]
Setup up a skeleton with Next.js that gets deployed and where we can start the work.

@smonolo i thought about it and I think the best solution is get rid of https://github.com/italia/next.developers.italia.it and start the work right in a subdirectory of this repo (fe. `nextjs/`), so we can better iterate.

That way we won't have to:
* setup a reverse proxy
* nor have client based redirect
* have to switch over everything once we are done
* use a less visible repo (https://github.com/italia/next.developers.italia.it) which can be confusing and potentially have less contributions

It's not ideal of course and kind of weird, but I think we can make it work.

**Comment by smonolo** [comment]
@bfabio Makes sense to me. I guess we can already get started with this issue as it's quite straight forward, and then see how it goes as we add port more stuff to the new project.

**Comment by bfabio** [comment]
Exactly, the other issues depend on this one 👍
